.circleci/config.yml

```
@@ -7,8 +7,9 @@ references:
       command: |
         apk add linux-headers
         apk add python3-dev
-        python3 -m venv venv
-        pip3 install -r requirements/base.txt
+        `which python3` -m venv venv
```

(nit) Using `` `which python3` `` doesn't add anything here.

```suggestion
        python3 -m venv venv
```

---

I have seen this method used a lot by more experienced python devs, although that is probably more for local use where they may have different versions and aliases i guess.